lib/__testfixtures__/object-prototype.input.mjs

```diff
@@ -0,0 +1,23 @@
+// Source from Object prototype
+const elt = document.createElement('div');
+elt.toString();
+elt.valueOf();
+elt.hasOwnProperty('remove');
+
+// Should not source from Object but TypeScript is behind
+// https://github.com/jackbsteinberg/get-originals-rewriter/issues/79
+const url = new URL();
+url.toString();
+const bool = new Boolean();
+bool.toString();
+const te = new TypeError();
+te.toString();
+
+// Don't source from Object prototype
+const n = new Number();
+n.valueOf();
```

Comment:
Can you test `const n2 = 5; n2.valueOf()` and also just `5..valueOf()`?

Reply:
Yes, I'm pretty sure these tests will fail, since the check for primitives goes before everything else.